stories/input.stories.js

```diff
@@ -9,38 +9,26 @@ export const input = () => (
     <Demo>
       <div>
         <div className="Manifold__Input">
-          <label for="my-input">Text input</label>
-          <input
-            type="text"
-            autoCapitalize="off"
-            name="text-input"
-            id="my-input"
-            placeholder="Placeholder"
-          />
```

Comment:
A lot of changes in this story were just making the HTML shorter & easier to read.